domain/src/main/java/me/androidbox/domain/agenda/model/Event.kt

```diff
@@ -0,0 +1,24 @@
+package me.androidbox.domain.agenda.model
+
+import me.androidbox.domain.alarm_manager.AgendaType
+
+data class Event(
+    override val id: String,
+    override val title: String,
+    override val description: String,
+    override val startDateTime: Long,
+    override val remindAt: Long,
+    override val agendaType: AgendaType = AgendaType.EVENT,
+    val endDateTime: Long,
+    val eventCreatorId: String,
+    val isUserEventCreator: Boolean,
+    val attendees: List<Attendee>,
+    val photos: List<String>,
+) : AgendaItem(
+    id = id,
+    title = title,
+    description = description,
+    startDateTime = startDateTime,
+    remindAt = remindAt,
+    agendaType = agendaType
+)
```

Comment:
Class for Events

Reply:
This is a personal opinion, but whenever I have a sealed class, I like to have all the childs in the same class

So in this case, Event, Reminder and Task data classes, on AgendaItem.kt